back/app/services/concerns/side_fx_helper.rb

```diff
@@ -33,4 +33,10 @@ def user_for_activity_on_anonymizable_item(item, user)
   def remove_user_from_past_activities_with_item(item, user)
     user.activities.where(item: item).update_all(user_id: nil)
   end
+
+  def sanitize_change(change)
+    change.delete(:updated_at)
+
+    change
+  end
```

Comment:
Not totally clear what this method is for - maybe a comment to explain what the sanitization is?